src/get_usernames.py

```diff
@@ -0,0 +1,60 @@
+import paramiko
+import getpass
+import argparse
+import sys
+import os
+from file_funcs import load_json
+
+def get_args():
+    argparser = argparse.ArgumentParser(description='Get usernames of students taking courses this semester')
+    argparser.add_argument('--username', '-u', help='Username for login', type=str)
+    argparser.add_argument('--password', '-p', help='Password for login', type=str)
+    argparser.add_argument('--semester', '-s', help='Semester to read data for', type=str)
```

Comment:
Misleading. You can only get usernames for current semester, so this option only decides where to read/write in local data.

> Semester to save data to

or similar.